drivers/usb/udc/udc_ameba.c

```diff
@@ -0,0 +1,897 @@
+/*
```

Comment:
Please open a separate PR to add the USB device controller driver and include information about the controller type. But first check if the controller type is already supported, for designs implemented by different vendors I would ask for a native driver implementation. At first glance it looks like this SoC has a DWC2 compatible controller, if so you need to use the udc_dwc2 driver.

Reply:
I have remove usb related code, and will open a separate PR in next commit to submit usb.